.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-1.9.3
```

Comment:
Version 1.9.3 is deprecated:
https://www.ruby-lang.org/en/news/2015/02/23/support-for-ruby-1-9-3-has-ended/

Abs,
Celso Vital Crivelaro

---------------------------------------------
Skype: celsocrivelaro


2015-07-09 14:14 GMT-03:00 Andre Souza <notifications@github.com>:

> In .ruby-version
> <https://github.com/locaweb/heartcheck/pull/6#discussion_r34278590>:
>
> > @@ -0,0 +1 @@
> > +ruby-1.9.3
>
> I think if we develop at lowest version supported is a good idea, this
> lock prevents the use of unavailable resources at 1.9.3
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/locaweb/heartcheck/pull/6/files#r34278590>.
>